llvm/include/llvm/ADT/Triple.h

```diff
@@ -215,7 +215,7 @@ class Triple {
     CoreCLR,
     Simulator,  // Simulator variants of other systems, e.g., Apple's iOS
     SYCLDevice,
-    LastEnvironmentType = Simulator
+    LastEnvironmentType = SYCLDevice
```

Comment:
I think it's better to put `SYCLDevice` before `Simulator`.
Less chances to conflict with future changes in this enum.

Reply:
Fine by me!